src/backend/lib/upgrade-download.js

```diff
@@ -225,14 +228,12 @@ class Download extends EventEmitter {
     http
       .get({ hostname: option.host, port: option.port, path: url }, res => {
         const filename = option.hash;
-        let sofar = 0;
-        const progress = through((chunk, enc, next) => {
-          sofar += chunk.length;
-          this.setState(UpgradeState.Download.Downloading, {
-            sofar,
-            total: option.size,
-          });
-          next(null, chunk);
+        const progress = progressStream({
+          length: option.size,
+          time: PROGRESS_THROTTLE, // ms between each progress event
```

Comment:
```suggestion
          time: PROGRESS_THROTTLE_MS, // ms between each progress event
```

Just saves us from having to specify 'ms' in every usage in a comment.